test/test/assets/screenshots/review.html

```diff
@@ -134,25 +134,50 @@
 const h2 = document.createElement('h2');
 document.body.appendChild(h2);
 
-// Remove the '?' at the front of the search string.
-const name = location.search.substr(1);
-if (!name) {
-  // Present a simple menu of screenshots to review.
-  h1.textContent = 'Select a screenshot to review:';
+const EXCLUDES = [
+  'native-cue-with-controls',
```

Comment:
Why is this constant defined in the code right here, rather than above with the definitions of `prefixes` and `screenshots`? Also, why is this one in all uppercase, while those ones are all lowercase?

Reply:
Fixed.  Thanks!